requirements.txt

```diff
@@ -14,6 +14,6 @@ Markdown==2.6.5
 # c2corg_common project
 # for development use a local checkout
 # -e ../v6_common
-git+https://github.com/c2corg/v6_common.git@343e1ef
+git+https://github.com/c2corg/v6_common.git@b8e402d
```

Comment:
The hash for c2corg_common with the fixes you suggested about some route/WP attributes is f825f8f